.github/workflows/cmake.yml

```diff
@@ -91,6 +93,8 @@ jobs:
     - name: Install Conan
       id: conan
       uses: turtlebrowser/get-conan@main
+      with:
+        version: 1.54.0
```

Comment:
Has this change gone back in by mistake from an older version

```suggestion
```

Reply:
Yes I think that was me branching off from the wrong branch, I'll remove it